docs/_data/meetups/usa-portland.yaml

```diff
@@ -0,0 +1,6 @@
+region: North America
+country: USA
+state: Oregon
+city: Portland
+notes: |
+  Organised by [@WriteTheDocsPDX](https://twitter.com/WriteTheDocsPDX)
```

Comment:
I lazily googled errors in sphinx, and expected to have access to it within conf.py